src/main/kotlin/no/nav/familie/ks/sak/kjerne/behandling/steg/StegService.kt

```diff
@@ -185,6 +189,18 @@ class StegService(
     // Denne metoden kalles av HentStatusFraOppdragTask for å sende behandling videre etter OK status fra oppdrag er mottatt
     fun utførStegEtterIverksettelseAutomatisk(behandlingId: Long) {
         val behandling = behandlingRepository.hentAktivBehandling(behandlingId)
+
+        // opprett tilbakekreving task om det finnes en tilbakekrevingsvalg
+        tilbakekrevingRepository.findByBehandlingId(behandlingId)?.let {
+            if (it.valg == Tilbakekrevingsvalg.IGNORER_TILBAKEKREVING) {
```

Comment:
kunne også snudd på flisa og gjort noen slik for å unngå return
``` 
if (it.valg != Tilbakekrevingsvalg.IGNORER_TILBAKEKREVING) {
               taskService.save(SendOpprettTilbakekrevingBehandlingRequestTask.opprettTask(behandlingId))
            } else {
logger.info(
                    """Tilbakekrevingsvalg er ${it.valg.name} for behandling $behandlingId.
                        Oppretter ikke tilbakekrevingsbehandling"""
                )
}      
```

Reply:
Bra forslag, fikser dette